.agents/skills/honcho-integration/references/bot-frameworks/nanobot/honcho_tool.py

```
@@ -0,0 +1,86 @@
+from nanobot.agent.tools.base import Tool
+
+
+class HonchoTool(Tool):
+    """
+    Tool for querying Honcho's AI-native memory.
+
+    Allows the agent to retrieve relevant context about users
+    based on their history and learned preferences.
+    """
+
+    def __init__(self, session_manager: "HonchoSessionManager"):
```

Ruff (0.15.20)

[error] Undefined name `HonchoSessionManager`

(F821)